internal/daemon/daemon.go

```diff
@@ -772,6 +774,11 @@ func (d *Daemon) Name() string {
 	return d.name
 }
 
+// Version ensures both the daemon and state have the same version.
```

Comment:
The comment feels misleading here as it's just a getter for the internal `.version` value?

Reply:
This follows the pattern used for `Name()`; since it's just a getter it hardly needs a comment (except for the linter...). I can add a commit to "refactor" these comments if that makes more sense here.